src/jvmTest/kotlin/html5-tags.kt

```diff
@@ -54,7 +54,8 @@ class Html5TagsTest {
                 <html>
                   <body>
                     <h1>header</h1>
-                    <div>content<span>yo</span>
+                    <div>
```

Comment:
This is caused by a change in some JDK's DOM implementation version. Not sure how do we eliminate it

Reply:
It might be a problem. But at least it works on TC so far...